### PR TITLE
Add USDT to banana mode and update code to reflect new storage

### PR DESCRIPTION
**Pull request by martin-chainflip**
# Pull Request

## Checklist

Please conduct a thorough self-review before opening the PR.

- [x] I am confident that the code works.
- [x] I have updated documentation where appropriate.

## Summary

This change adds USDT to the currencies supported by banana mode.
It also updates the banana mode code to support the new storage locations introduced as part of the 1.3 update.

### CODE REVIEW

File: bouncer/commands/go_bananas.ts [hunk 1]
```diff
@@ -39,9 +40,14 @@ type AmountChange = null | {
...
+type ComplexAsset = {
+  chain: string;
+  asset: string;
+};
+
 type LimitOrderResponse = {
-  base_asset: string;
-  quote_asset: string;
+  base_asset: string | ComplexAsset;
+  quote_asset: string | ComplexAsset;
```
Comment (kylezs) [inline bouncer/commands/go_bananas.ts:50]: why do we need two types here? is this to support legacy and new or?

Comment (albert-llimos) [reply bouncer/commands/go_bananas.ts:50]: I can clean this up once I merge #4668 where I have done the heavy lifting to update all the assets.

Comment (martin-chainflip) [reply bouncer/commands/go_bananas.ts:50]: yes, the asset could be "USDC" or {chain: "Ethereum", asset: "USDT"}

Comment (kylezs) [reply bouncer/commands/go_bananas.ts:50]: Yeah ok, as long we're tracking it and plan to remove it soon

**Review by kylezs — APPROVED** [review]

**Review by albert-llimos — APPROVED** [review]

**Pull request merged**